### PR TITLE
Add instructions for `apt-get` install to `README`

### PR DESCRIPTION
**Pull request by ldennington**
Updating `README.md` with instructions for `apt-get` setup and install
for Ubuntu Bionic and Hirsute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,9 +112,33 @@ Or you can run the `git update-microsoft-git` command, which will run those brew
...
 ## Linux
 
-For Ubuntu/Debian distributions, `apt-get` support is coming soon. For now, please use the most
-recent [`.deb` package](https://github.com/microsoft/git/releases). For example, you can download a
-specific version as follows:
+`apt-get` support is available for Ubuntu Bionic Beaver (18.04) and Hirsute 
+Hippo (21.04). Take the following steps to set up and install based on the
+version you are running:
+
+### Bionic
```
Comment (derrickstolee) [inline README.md]: Perhaps list this subsection as

```suggestion
### Ubuntu 18.04 (Bionic)
```

Similarly, `Ubuntu 21.04 (Hirsute)` in the one below.

File: README.md [hunk 2]
```diff
@@ -112,9 +112,33 @@ Or you can run the `git update-microsoft-git` command, which will run those brew
...
+
+```shell
+curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+sudo apt-add-repository https://packages.microsoft.com/ubuntu/18.04/prod
+sudo apt-get update
+sudo apt-get install microsoft-git
```
Comment (derrickstolee) [inline README.md:125]: Did we update the action to publish as `microsoft-git`, or is it still `git-vfs`? Can that be part of this PR?

Comment (ldennington) [reply README.md:125]: No updates to the pipeline are required to publish under a new name. However, there is one small clarifying tweak I want to make, which I will include as a separate commit in this PR. Stay tuned!

**Review by derrickstolee — APPROVED** [review]
LGTM. [Pushing the `microsoft-git` version now](https://github.com/microsoft/git/actions/runs/990944865).

**Pull request merged**